src/ess/dream/io/geant4.py

```diff
@@ -90,10 +91,14 @@ def group(key: str, da: sc.DataArray) -> sc.DataArray:
         if key in ["high_resolution", "sans"]:
             # Only the HR and SANS detectors have sectors.
             res = da.group("sector", *elements)
+        elif key in ["endcap_backward", "endcap_forward"]:
+            # Other banks only have a single SUMO.
+            res = da.group("sumo", *elements)
```

Comment:
Even if other banks only have a single SUMO, we can still group them by `SUMO`, no...?

Reply:
We can. But that would be the use? This extra dimension would get in the way of plotting and would always have to be sliced out.

Reply:
Yeah, I wasn't sure how many times we have to filter the endcap for this reason.
If it's not in many places, I think not-grouping is fine.